reference/functions.md

```diff
@@ -1311,7 +1313,7 @@ Introduced in: **Clarity 3**
 The `get-stacks-block-info?` function fetches data for a block of the given *Stacks* block height. The value and type returned are determined by the specified `StacksBlockInfoPropertyName`. If the provided `stacks-block-height` does not correspond to an existing block prior to the current block, the function returns `none`. The currently available property names are as follows:
 
 - `id-header-hash`: This property returns a `(buff 32)` value containing the _index block hash_ of a Stacks block.   This hash is globally unique, and is derived from the block hash and the history of accepted PoX operations.  This is also the block hash value you would pass into `(at-block)`.
-- `header-hash`: This property returns a `(buff 32)` value containing the header hash of a Stacks block, given a Stacks chain height. **WARNING** this hash is not guaranteed to be globally unique, since the same Stacks block can be mined in different PoX forks.  If you need global uniqueness, you should use `id-header-hash`.
+- `header-hash`: This property returns a `(buff 32)` value containing the header hash of a Stacks block, given a Stacks chain height.
```

Comment:
I think this one is okay to leave. There is one that should be removed in `stacks-block-height`.

Reply:
54ac603ffbdf59908c43c32c78cfb852d25980e9